x/gov/module.go

```diff
@@ -186,7 +186,7 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 }
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (am AppModule) ConsensusVersion() uint64 { return 3 }
+func (am AppModule) ConsensusVersion() uint64 { return 4 }
```

Comment:
Just adding this would cause error.

Reply:
Done, no data migration may not need to change consensusVersion